src/Microsoft.DotNet.ProjectJsonMigration/Microsoft.DotNet.Internal.ProjectModel/AnalyzerOptions.cs

```diff
@@ -3,9 +3,9 @@
 
 using System;
 
-namespace Microsoft.DotNet.ProjectModel
+namespace Microsoft.DotNet.Internal.ProjectModel
 {
-    public class AnalyzerOptions : IEquatable<AnalyzerOptions>
+    internal class AnalyzerOptions : IEquatable<AnalyzerOptions>
```

Comment:
Is this just a temporary stepping stone?  I thought the plan was to use Preview2 ProjectModel.